src/main/java/uk/gov/hmcts/reform/idam/client/models/ExchangeCodeRequest.java

```diff
@@ -1,20 +1,22 @@
 package uk.gov.hmcts.reform.idam.client.models;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import feign.form.FormProperty;
 import lombok.Builder;
 import lombok.EqualsAndHashCode;
+import lombok.Getter;
 
 @EqualsAndHashCode
 @Builder
+@Getter
 public class ExchangeCodeRequest {
     private String code;
-    @JsonProperty("grant_type")
+    @FormProperty("grant_type")
     private String grantType;
-    @JsonProperty("redirectUri")
+    @FormProperty("redirect_uri")
```

Comment:
redirectUri -> redirect_uri is this fine ?

Reply:
it should be. in the swagger it is redirect_uri.
I guess idam ignores unknown